package proto_ff;

import "yd_fieldoptions.proto";


message E_GunvalueConfig
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "唯一ID"];
	optional int32 m_value = 2[(yd_fieldoptions.field_cname) = "值"];
	optional int32 m_roomid = 3[(yd_fieldoptions.field_cname) = "房间ID"];
	optional int32 m_gunid = 4[(yd_fieldoptions.field_cname) = "炮Id"];
}


message Sheet_GunvalueConfig
{
	repeated E_GunvalueConfig E_GunvalueConfig_List = 1[(yd_fieldoptions.field_arysize)=100];
}
